capstone.pc.in

```diff
@@ -1,7 +1,12 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${prefix}/lib@LIBSUFFIX@
+includedir=${prefix}/include
```

Comment:
this line should be `includedir=${prefix}/include/capstone`, no?

Reply:
It will work either way, because it automatically looks for project name folder inside that path, it wont hurt to specify the whole path, like you did... so either way.

Reply:
an example is here: https://mailman.videolan.org/pipermail/x265-devel/2013-October/001697.html

Reply:
here is another one: https://sourceforge.net/p/cplusplus/libzrtpcpp/ci/master/tree/libzrtpcpp.pc.cmake